GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -157,7 +157,9 @@ class ChatViewModel: EngagementViewModel, ViewModel {
             // messages from operator until message is sent from visitor.
             self?.interactor.withConfiguration { [weak self] in
                 guard let self = self else { return }
-                if case .startEngagement = self.startAction, self.chatStorageState().isEmpty() {
+                if
```

Comment:
Do you think that putting the condition in a `let` and then asking `if condition` would help make this code more readable? Although I don't know if you can do that with `case`. Right now it's not clear to me what scenario we are evaluating.